app/src/main/java/ua/pp/marchik/root/myapplication/MainActivity.java

```diff
@@ -28,6 +28,7 @@ public void onClick(View view) {
         });
     }
 
+
     @Override
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://77.121.149.190:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)